test/plugin/test_out_google_cloud.rb

```diff
@@ -463,6 +482,35 @@ def stub_metadata_request(metadata_path, response_body)
                 :headers => {'Content-Length' => response_body.length})
   end
 
+  def setup_auth_stubs
+    # Used by 'googleauth' to test whether we're running on GCE.
+    # It only cares about the request succeeding with Metdata-Flavor: Google.
+    stub_request(:get, 'http://169.254.169.254').
```

Comment:
Would it be useful to add that comment in the code? (meaning that this IP == http://metadata/)